roverrobotics_input_manager/scripts/ps4_mapper.py

```diff
@@ -71,35 +80,39 @@ def cb_status(self, msg):
         if (msg.button_l1 or msg.button_r1) and self.buttonpressed is False:
             trim_msg = Float32()
             if msg.button_r1:
-                trim_msg = -self._trim_incre_value
+                trim_msg = .01
```

Comment:
This file should be totally rewritten at some point since it is incredibly hard to follow and there are magic numbers all over the place.